README.md

```diff
@@ -1,18 +1,85 @@
 # NAME
 
-WebService::MinFraud - Perl API for MaxMind's minFraud web services
+WebService::MinFraud - Perl API for MaxMind's minFraud Score and Insights web services
 
 # VERSION
 
 version 0.001001
 
+# SYNOPSIS
+
+    use 5.010;
+
+    use WebService::MinFraud::Client;
+
+    # The Client object can be re-used across several requests.
+    # Your MaxMind user_id and license_key are available at
+    # https://www.maxmind.com/en/my_license_key
+    my $client = WebService::MinFraud::Client->new(
+        user_id     => '42',
+        license_key => 'abcdef123456',
+    );
+
+    # Request HashRef must contain a 'device' key, with a value that is a
+    # HashRef containing an 'ip_address' key with a valid IPv4 or IPv6 address.
+    # All other keys/values are optional; see other modules in minFraud perl API
+    # distribution for details.
+
+    my $request = { device => { ip_address => '24.24.24.24' } };
+
+    # Use the 'score' or 'insights' client methods, depending on the minFraud
+    # web service you are using.
+
+    my $score = $client->score( $request );
+    say $score->risk_score;
+
+    my $insights = $client->insights( $request );
+    say $insights->shipping_address->is_high_risk;
+
+# DESCRIPTION
+
+This distribution provides an API for the
+[MaxMind minFraud Score and Insights web services](http://dev.maxmind.com/minfraud/minfraud-score-and-insights-api-documentation/).
+
+See [WebService::MinFraud::Client](https://metacpan.org/pod/WebService::MinFraud::Client) for details on using the web service client
+API.
+
+# INSTALLATION
+
+The minFraud perl API and its dependencies can be installed with
+[cpanm](https://metacpan.org/pod/App::cpanminus).  `cpanm` itself has no
+dependencies.
+
+    cpanm WebService::MinFraud
+
+# VERSIONING POLICY
+
+The minFraud perl API uses [Semantic Versioning](http://semver.org/).
+
+# PERL VERSION SUPPORT
+
+The minimum required perl version for the minFraud perl API is 5.10.0.
```

Comment:
Capitalized Perl is generally used unless one is specifically talking about the interpreter.